### PR TITLE
Changed dns displaying, using a textarea

### PR DESCRIPTION
**Pull request by MMaiero**
Signed-off-by: Maiero <matteo.maiero@eurotech.com>

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
Hey @MMaiero,
I have checked this and it looks fine, I just have 2 comments.
1. Error message should be changed if user has e.g. 15 DNS addresses and one is wrong (Kura reports "Invalid IPv4 Address", which is not correct, it should really say "one of ip addresses not valid, please check" or something similar. But this is just cosmetic
Screenshot 2020-06-19 at 14 49 54

2. The field is not limited - but there should be some sort of limitation.

**Comment by MMaiero** [comment]
What do you mean with 

> but there should be some sort of limitation.

**Comment by LeoNerdoG** [comment]
I meant that there is no upper limit for number of character user can enter into the field. If there is no limitation, one can break the system by entering 10M+ characters into the field and applying the changes. 
So I mean that there should be a limit for e.g. 10000 characters (that is almost 700 full-length DNS addresses, e.g. 255.255.255.255).

**Comment by MMaiero** [comment]
I have to see if there are limits set at linux level
Looking at my Mac there seem to be no limit. But I'll double check

**Comment by pierantoniomerlino** [comment]
Indeed, there aren't any limit on the number of dns servers that can be added to the  resolv.conf file.

**Review by pierantoniomerlino — APPROVED** [review]

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabTcpIpUi.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
 import org.gwtbootstrap3.client.ui.ListBox;
 import org.gwtbootstrap3.client.ui.Modal;
 import org.gwtbootstrap3.client.ui.PanelHeader;
+import org.gwtbootstrap3.client.ui.TextArea;
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabTcpIpUi.java:45]: Hey @MMaiero,
Can you please update the header to "2020"?

**Comment by MMaiero** [comment]
Fixed the copyright header and the message, as suggested.

**Review by pierantoniomerlino — APPROVED** [review]

**Pull request merged**